platforms/documentation/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc

```diff
@@ -126,9 +186,9 @@ repositories {
 }
 
 dependencies {
-    testImplementation("org.junit.jupiter:junit-jupiter:5.9.3")     // <3>
+    testImplementation(libs.junit.jupiter)                          // <3>
```

Comment:
Here the version catalog is being referenced but I dont see it in the layout above